src/cider/nrepl/middleware/test.clj

```diff
@@ -64,7 +64,8 @@
      (->> e
           .getStackTrace
           (map (partial st/analyze-frame namespaces))
-          (filter #(= (:class %) class-name))
+          (filter
+           #(clojure.string/starts-with? (:class %) (.getName (class f))))
```

Comment:
I'd use some-> out of caution

(`=` doesn't fail on nils, `starts-with?` does)